srcpkgs/ghdl/template

```diff
@@ -63,9 +63,15 @@ do_configure() {
 		cd ..
 	fi
 	if [ "$build_option_llvm" ]; then
+		# FIXME: This is a bit hacky. This should be replaced by the path within the gcc
+		# source tree for libbacktrace when building the gcc backend has been added.
+		gcc_dir=$(gcc --print-search-dirs | awk '{ if ($1 == "install:") print $2 }')
```

Comment:
Doing this like we do in `python3-pyside` is better. You can just build the library path yourself.

`		local gcc_ver="$(${XBPS_CROSS_TRIPLET}-gcc -dumpversion)"`

Reply:
Ah neat. Thanks for that.